content/php/concepts/math-functions/terms/hexdec/hexdec.md

```diff
@@ -0,0 +1,56 @@
+---
+Title: 'hexdec()'
+Description: 'Returns the decimal equivalent of a given hexadecimal string.'
+Subjects:
+- 'Computer Science'
+- 'Web Development'
+Tags:
+- 'Arithmetic'
+- 'Encoding'
+- 'Functions'
+- 'Math'
+CatalogContent:
+- 'learn-php'
+- 'paths/computer-science'
+---
+
+The **`hexdec()`** function returns the decimal equivalent of a given hexadecimal string. `hexdec()` ignores any non-hexadecimal characters,
+computing the result as if they did not exist.
+
+`hexdec()` returns a value of type `int` if the converted number fits into the platform's `int` type; otherwise, a `float` is returned.
+
+## Syntax
+
+```pseudo
+$decimal = hexdec(hexString);
+```
+
+Where `hexString` is the hexadecimal string that is to be converted into a decimal.
+
+## Example
+
+This example converts the hexadecimal string `"CDECADE"` into its decimal equivalent.
+
+```php
+$decimal = hexdec("CDECADE");
+echo $decimal;
+```
```

Comment:
```suggestion
```php
<?php
  $decimal = hexdec("CDECADE");
  echo $decimal;
?>
```
```

Reply:
@caupolicandiaz I have implemented this change in the most recent commit.